heroesfighter/src/Components/Card.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import contactCard from './ContactCard.jsx';
+import './Card.css'
+
+class Card extends React.Component{
```

Comment:
Same as the other component : if stateless => functional 
And even if you don't have ESLint unuse spaces.